.travis.yml

```diff
@@ -3,6 +3,6 @@ language: python
 python:
   - "3.3"
 # command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: "pip install pep8"
+install: "pip install pep8 dnspython"
```

Comment:
This would also be required in https://github.com/freifunk/icvpn-meta/blob/master/.travis.yml before this PR is merged to ensure the checks will run.